bats/tests/helpers/vm.bash

```diff
@@ -179,7 +179,7 @@ start_application() {
 launch_the_application() {
     local args=("$@")
 
-    if using_dev_mode; then
+    if using_dev_mode && ! is_windows; then
```

Comment:
This is not necessary; `using_dev_mode` cannot be true on Windows: https://github.com/rancher-sandbox/rancher-desktop/blob/3a05844c91a50f859f606478de8f59b2bf4f762b/bats/tests/helpers/paths.bash#L36-L38

Once could argue that the error message should say `yarn` instead of `npm` though.